### ISSUE TITLE
NetworkManager: Error: Failed to add '<ssid>' connection: Insufficient privileges.

### ISSUE THREAD

**Issue by obadz** [NixOS/nixpkgs#9532]
Doing

```
$ nmcli con add con-name <ssid> ifname wlp1s0 type wifi ssid <ssid>
```

or using the nm-applet, I get the "Insufficient privileges" error.

I'm a member of the networkmanager (gid = 57) group and I'm running network manager with:

```nix
networking.networkmanager.enable = true;
```

Nothing in the log.  Used to work fine with GNOME3.

Cc @lethalman @edolstra

**Comment by domenkozar** [comment]
what branch / nixos version?

**Comment by obadz** [comment]
Previous nixos-unstable release: 69367e4ebcacd65b8e4fce52c80c7d6bda20c404

**Comment by domenkozar** [comment]
@obadz can you try release-15.09?

**Comment by obadz** [comment]
Somehow, I can no longer reproduce even with the same version I originally had used.  Was anyone able to reproduce?

**Comment by domenkozar** [comment]
still an issue somehow?

**Comment by domenkozar** [comment]
well reopen if it appears again

**Comment by domenkozar** [comment]
It happend to me on 15.09

**Comment by domenkozar** [comment]
Fixed in 4c2bbb248cf22ad3c3541ba7d38bbc3abb40c706
